libcontainer/cgroups/systemd/common.go

```diff
@@ -21,6 +22,10 @@ var (
 	connOnce sync.Once
 	connDbus *systemdDbus.Conn
 	connErr  error
+
+	versionOnce sync.Once
```

Comment:
Note that all this sync.Once stuff is currently redundant, as we only call systemdVersion() once. It was added in anticipation there will be other systemd parameters like this that need a version guard, and since we call `systemdVersion()` when and where needed (right from `addCpuQuota`) rather than in advance, the future calls might come from a similar "when and where needed" places.